descriptors/ModuleDescriptor-template.json

```diff
@@ -25,6 +28,9 @@
           "methods": [
             "POST"
           ],
+          "modulePermissions": [
+          "consortia.job.collection.get"
+          ],
```

Comment:
Why for POST Tenant we need permission to get job collections?

Reply:
fixed